atom/browser/native_window.cc

```diff
@@ -699,12 +699,24 @@ bool NativeWindow::OnMessageReceived(const IPC::Message& message) {
   IPC_BEGIN_MESSAGE_MAP(NativeWindow, message)
     IPC_MESSAGE_HANDLER(AtomViewHostMsg_UpdateDraggableRegions,
                         UpdateDraggableRegions)
+    IPC_MESSAGE_HANDLER(AtomViewHostMsg_ToggleFullscreen,
+                        ToggleFullscreen)
     IPC_MESSAGE_UNHANDLED(handled = false)
   IPC_END_MESSAGE_MAP()
 
   return handled;
 }
 
+void NativeWindow::ToggleFullscreen(bool enter_fullscreen) {
```

Comment:
We should not use `ToggleFullscreen` as method name, it sounds like a `NativeWindow` API, but it is actually a IPC message handler. `ToggleFullScreenFromRenderer` could be a better name. And the definition of this method should be placed near the `UpdateDraggableRegions`.

Also we tend to use `FullScreen` instead of `Fullscreen` in method names to be consistent with Chromium.